composer.json

```diff
@@ -210,24 +210,24 @@
                 "4033 : Turn off memcache on ODEs, hack for our issue 4033": "patches/acquia/memcache.settings-hack-for-4033.patch"
             },
             "drupal/core": {
-                "2952990 : Missing schema for display_options.defaults.sitename_title": "https://www.drupal.org/files/issues/2018-03-14/views-schema-2952990-2.patch",
+                "2952990 : Missing schema for display_options.defaults.sitename_title (https://www.drupal.org/project/drupal/issues/2952990#comment-12526074, comment 2)": "patches/drupal/core/2952990-2-views-schema.patch",
                 "3150294 : (Unofficial Patch) New translations should use initial workflow state" : "patches/drupal/workflow-hack-for-3150294.patch",
-                "1308152 : Add stream wrappers to access extension files (https://www.drupal.org/project/drupal/issues/1308152#comment-15653396)": "patches/drupal/core/1308152-404-stream-wrapper.patch",
-                "2971390 : Make exposure of translation meta fields conditional (https://www.drupal.org/project/drupal/issues/2971390#comment-15882642)" : "patches/drupal/core/2971390-105-translation-meta-field-conditional-exposure.patch",
-                "3207813 : ModuleHandler skips all hook implementations when invoked before the module files have been loaded (https://www.drupal.org/project/drupal/issues/3207813#comment-14962228, comment #27)": "patches/drupal/core/wokraround-patch-for-3207813.patch"
+                "1308152 : Add stream wrappers to access extension files (https://www.drupal.org/project/drupal/issues/1308152#comment-15653396, comment 404)": "patches/drupal/core/1308152-404-stream-wrapper.patch",
```

Comment:
Comment not found, you say? Why'd you even bother including it?

Reply:
Such a corny comment, you'd think **I** wrote it for goodness sake!